source/fe/mapping_q_eulerian.cc

```diff
@@ -130,8 +131,9 @@ MappingQEulerian<dim, VectorType, spacedim>::get_vertices(
     dynamic_cast<const MappingQEulerianGeneric &>(*this->qp_mapping)
       .compute_mapping_support_points(cell);
 
-  std::array<Point<spacedim>, GeometryInfo<dim>::vertices_per_cell>
-    vertex_locations;
+  boost::container::small_vector<Point<spacedim>,
+                                 GeometryInfo<dim>::vertices_per_cell>
+    vertex_locations(GeometryInfo<dim>::vertices_per_cell);
   std::copy(a.begin(),
             a.begin() + GeometryInfo<dim>::vertices_per_cell,
             vertex_locations.begin());
```

Comment:
I think this should work:
```suggestion
  boost::container::small_vector<Point<spacedim>,
                                 GeometryInfo<dim>::vertices_per_cell>
    vertex_locations(a.begin(),
                     a.begin() + GeometryInfo<dim>::vertices_per_cell);
```